app/models/campaign.rb

```diff
@@ -86,6 +88,21 @@ def ensure_correct_campaign_attributes
     end
   end
 
+  def validate_url
+    return unless trigger_rules['url']
+
+    errors.add(:url, 'invalid') if inbox.inbox_type == 'Website' && !url_valid?(trigger_rules['url'])
```

Comment:
Better to have a proper error msg instead of just `invalid`.

Reply:
@aswindevps The final response should be like this
![Screenshot 2022-02-04 at 2 10 28 PM](https://user-images.githubusercontent.com/12408980/152498055-b90e4a5f-ba88-49fd-be0c-47d411bdccd6.png)